src/include/benchmark/tpcc/tpcc_workload.h

```diff
@@ -51,7 +51,9 @@ void ExecuteUpdate(executor::AbstractExecutor* executor);
 
 void ExecuteDelete(executor::AbstractExecutor* executor);
 
-void PinToCore(size_t core);
+#ifndef __APPLE__ // https://superuser.com/questions/149312/how-to-set-processor-affinity-on-os-x
+    void PinToCore(size_t core);
+#endif
```

Comment:
Instead of commenting it out in the header, can you leave it in and pound-define it out the CPP implementation? Less noise in the code that way imho.